resources/js/Pages/Collections/Components/FeaturedCollections/FeaturedCollectionsItem.tsx

```diff
@@ -0,0 +1,158 @@
+import cn from "classnames";
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { ButtonLink } from "@/Components/Buttons/ButtonLink";
+import { GridHeader } from "@/Components/GridHeader";
+import { Heading } from "@/Components/Heading";
+import { Img } from "@/Components/Image";
+import { NetworkIcon } from "@/Components/Networks/NetworkIcon";
+import { CollectionNft } from "@/Pages/Collections/Components/CollectionNft";
+import { FormatCrypto } from "@/Utils/Currency";
+
+const truncateDescription = (
+    description: App.Data.Collections.CollectionFeaturedData["description"],
+): string | null => {
+    if (description !== null && description.length > 120) {
+        return description.slice(0, 120) + "...";
+    }
```

Comment:
personally I'd change the check as follows:
```suggestion
    if (description !== null && description.length > 120) {
        return description.slice(0, 117) + "...";
    }
```

that way you don't end up with 120 characters + 3 additional ones or the weird scenario where you'd have a 121-123 character description that would be truncated but take up the same amount of width due to the `...` being added